typesafety/tests/externalmodule.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2013-2015 BalaBit
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+
+# External module whose objects should not be decorated when another
+# module imports some of its objects.
+
+
+class UndecoratedClass:
```

Comment:
Differently how? We don't actually trip over annotations we don't understand, we just silently ignore them (as to not trip over annotation systems used parallel to typesafety). If we don't need to play nice to this depth, we can remove it, but that's a separate issue.